main.go

```diff
@@ -170,18 +170,18 @@ func main() {
 		LeaderElectionID:        "multicloudhub-operator-lock",
 		WebhookServer:           &ctrlwebhook.Server{TLSMinVersion: "1.2"},
 		LeaderElectionNamespace: ns,
-		LeaseDuration:          &leaseDuration,
-		RenewDeadline:          &renewDeadline,
-		RetryPeriod:            &retryPeriod,
+		LeaseDuration:           &leaseDuration,
+		RenewDeadline:           &renewDeadline,
+		RetryPeriod:             &retryPeriod,
 	}
 
-	cacheSecrets := os.Getenv(NoCacheEnv)
-	if len(cacheSecrets) > 0 {
-		setupLog.Info("Operator Client Cache Disabled")
-		mgrOptions.ClientDisableCacheFor = []client.Object{
-			&corev1.Secret{},
-			&olmv1alpha1.ClusterServiceVersion{},
-		}
+	mgrOptions.ClientDisableCacheFor = []client.Object{
+		&corev1.Secret{},
+		&rbacv1.ClusterRole{},
+		&rbacv1.ClusterRoleBinding{},
+		&rbacv1.RoleBinding{},
+		&corev1.ConfigMap{},
```

Comment:
Does the [configmap watch](https://github.com/stolostron/multiclusterhub-operator/blob/main/controllers/multiclusterhub_controller.go#L537) still work with configmaps being disabled?

Reply:
Comment from Erin today was that everything worked even with the disablement.